pandas/core/indexes/multi.py

```diff
@@ -811,7 +811,9 @@ def duplicated(self, keep='first'):
 
     @Appender(ibase._index_shared_docs['fillna'])
```

Comment:
should this line be removed?

Reply:
I don't think so.  That appends additional documentation.

Reply:
Ah, but I see #18018.  In that case, leave this alone for this PR.  Make the change in a subsequent PR.

Reply:
Moved `fillna` changes to PR #18269 .